libmate-panel-applet/mate-panel-applet.c

```diff
@@ -711,6 +711,46 @@ mate_panel_applet_setup_menu_from_file (MatePanelApplet    *applet,
 	g_free (xml);
 }
 
+/**
+ * mate_panel_applet_setup_menu_from_resource:
+ * @applet: a #MatePanelApplet.
+ * @resource_path: a resource path
+ * @action_group: a #GtkActionGroup.
+ *
+ * Sets up the context menu of @applet. @filename is a resource path to a menu
+ * XML file, containing a #GtkUIManager UI definition that describes how to
+ * display the menu items. @action_group contains the various #GtkAction that
+ * are referenced in @xml.
+ *
+ * See also the <link linkend="getting-started.context-menu">Context
+ * Menu</link> section.
+ *
+ * Since: 3.4
```

Comment:
This is not gnome-panel, we don't have 3.4 :slightly_smiling_face:

Reply:
fixed